domains/potatopioneers.is-a.dev

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "potatoPioneer's website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
+        "gitlab": "ponfertato"
+        "telegram": "ponfertato"
+        "vk": "ponfertato"
+        "discord": "ponfertato"
+    },
+    "record": {
+        "CNAME": "potatopioneers.github.io/potatopioneers.site",
```

Comment:
You can't use a CNAME with other records. Also, your CNAME is invalid.

Reply:
Change the CNAME record to the address provided by Cloudflare